src/megatron/bridge/models/qwen_vl/modelling_qwen3_vl/transformer_block.py

```
@@ -29,6 +29,7 @@
 from megatron.core.packed_seq_params import PackedSeqParams
 from megatron.core.transformer.transformer_block import TransformerBlock
 from megatron.core.utils import WrappedTensor, deprecate_inference_params, make_viewless_tensor
+from megatron.core.process_groups_config import ProcessGroupCollection
```

Flake8 (7.3.0)

[error] 'megatron.core.process_groups_config.ProcessGroupCollection' imported but unused

(F401)